scripts/o3de/o3de/android.py

```diff
@@ -0,0 +1,598 @@
+        else:
+            signing_config = None
+
+        if signing_config is not None:
+            logger.info("Signing configuration enabled")
+        else:
+            logger.warning("No signing configuration enabled. This output APK will not be signed until a signing configuration is added.")
+
+        apg = android_support.AndroidProjectGenerator(engine_root=engine_path,
+                                                      android_build_dir=build_folder,
+                                                      android_sdk_path=sdk_manager.get_android_sdk_path(),
+                                                      android_build_tool_version=sdk_build_tools_version,
+                                                      android_platform_sdk_api_level=android_platform_sdk_api_level,
+                                                      android_ndk_package=ndk_package,
+                                                      project_name=project_name,
+                                                      project_path=project_path,
+                                                      project_general_settings=project_settings,
+                                                      project_android_settings=android_settings,
+                                                      cmake_version=cmake_version,
+                                                      cmake_path=cmake_path,
+                                                      gradle_path=gradle_path,
+                                                      gradle_version=gradle_version,
+                                                      gradle_custom_jvm_args=custom_jvm_args,
+                                                      android_gradle_plugin_version=android_gradle_plugin_ver,
+                                                      ninja_path=ninja_path,
+                                                      asset_mode=args.asset_mode,
+                                                      signing_config=signing_config,
+                                                      extra_cmake_configure_args=extra_cmake_args,
+                                                      overwrite_existing=True,
+                                                      strip_debug_symbols=strip_debug,
+                                                      src_pak_file_path=src_bundle_pak_subfolder,
+                                                      oculus_project=is_oculus_project)
+        apg.execute()
+
+    except android_support.AndroidToolError as err:
+        logger.error(str(err))
+        return 1
+    else:
+        return 0
+
+
+def add_args(subparsers) -> None:
+    """
+    add_args is called to add subparsers arguments to each command such that it can be
+    a central python file such as o3de.py.
+    It can be run from the o3de.py script as follows
+    call add_args and execute: python o3de.py register --engine-path "C:/o3de"
```

Comment:
This comment should be updated to mention `android` command instead of the `register` command.
```suggestion
    call add_args and execute: python o3de.py android"
```